### PR TITLE
Integration test for work anniversary with employee service

### PR DESCRIPTION
**Pull request by rohanj-qxf2**

### CODE REVIEW

File: work-anniversary/tests/test_integration_work_anniversary_employee.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import sys
+import boto3
+from unittest import mock
+from unittest.mock import patch
+from work_anniversary import work_anniversary
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+@patch('work_anniversary.get_all_employees',return_value=[{'node':{'email':'test@qxf2.com','firstname':'Test','lastname':'Qxf2','dateJoined':'03-Feb-2013','isActive':'Y'}}])
+@patch('work_anniversary.credentials.PASSWORD', 'new_value') 
+def test_work_anniversary_employee_service(mock_get_all_employees):
```
Comment (Annapooraniqxf2) [inline work-anniversary/tests/test_integration_work_anniversary_employee.py:15]: Doc string for the funtion is missing

File: work-anniversary/tests/test_integration_work_anniversary_employee.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+import os
+import sys
+import boto3
+from unittest import mock
+from unittest.mock import patch
+from work_anniversary import work_anniversary
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+@patch('work_anniversary.get_all_employees',return_value=[{'node':{'email':'test@qxf2.com','firstname':'Test','lastname':'Qxf2','dateJoined':'03-Feb-2013','isActive':'Y'}}])
```
Comment (Annapooraniqxf2) [inline work-anniversary/tests/test_integration_work_anniversary_employee.py:13]: Return value can be taken from the conf

File: work-anniversary/tests/test_integration_work_anniversary_employee.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+@patch('work_anniversary.credentials.PASSWORD', 'new_value')
```
Comment (Annapooraniqxf2) [inline work-anniversary/tests/test_integration_work_anniversary_employee.py:14]: we can use 'as' and we can shorten the lengthy usage of variables
Where are we using this 'new-value'

**Comment by akkuldn** [comment]
This PR will not be merged as the test does not check any functionality  or component of the Work anniversary lambda.
Hereby closing this PR

**Pull request closed without merging**